llvm/lib/Target/SPIRV/SPIRVCommandLine.cpp

```diff
@@ -119,6 +119,13 @@ bool SPIRVExtensionsParser::parse(cl::Option &O, StringRef ArgName,
       continue;
     }
 
+    if (Token == "KHR") {
```

Comment:
is the token normalized to upper? Shall we accept both `KHR` and `khr`?

Reply:
I can add it if you want.

Reply:
I'd say that's a better user experience and it doesn't cost much